src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```

Comment:
Other useful methods could also be included in the `Contact` class, such as `isValid`, the constructor since many subclasses of `Contact` seem to have the signature `Class(String value, boolean isPrivate)`. Otherwise, what's the point? 😛